### PR TITLE
Added links + video + changes from Adam

### PR DESCRIPTION
**Pull request by tiagov8**

### CODE REVIEW

File: rules/defining-pbis/rule.md [hunk 1]
```diff
@@ -11,45 +11,43 @@ related: []
 created: 2022-10-18T03:27:00.098Z
 guid: da1579f4-b598-4a2c-8fed-0b9b2230f6e8
 ---
-It is very common that a developer jumps onto a team and is given a PBI to work on, only to find out that it has almost no information. When that happens, it is crucial for the developer to raise their voice and gather enough information to be able to work on it.
+It is very common that a developer looks at a PBI to work on, only to find out that it has almost no information. When that happens, it is crucial for the developer to raise their voice and gather enough information so it meets the Definition of Ready.
```
Comment (pierssinclairssw) [inline rules/defining-pbis/rule.md:14]: can you add a link to definition of ready and  remove link further down

File: rules/defining-pbis/rule.md [hunk 2]
```diff
@@ -11,45 +11,43 @@ related: []
...
 Generally, there are a few pieces of information that every PBI should have:
 
 * Title
-* Description
-
-  * The required steps and critical information to complete the PBI
-* Acceptance Criteria
-
+* [Description](/spec-do-you-use-user-stories)
+  * The meat of the PBI
```
Comment (pierssinclairssw) [inline rules/defining-pbis/rule.md:24]: this has overwritten tylahs change

**Review by pierssinclairssw — CHANGES_REQUESTED** [review]

File: rules/defining-pbis/rule.md (hunk 1 above)
Comment (tiagov8) [reply rules/defining-pbis/rule.md]: done

File: rules/defining-pbis/rule.md (hunk 2 above)
Comment (tiagov8) [reply rules/defining-pbis/rule.md]: I thought it was intended as Adam sent the content to me. I added back the old sentence there

**Review by pierssinclairssw — APPROVED** [review]

**Pull request merged**